spec/compiler/codegen/debug_spec.cr

```diff
@@ -252,4 +252,12 @@ describe "Code gen: debug" do
       LibFoo.foo = ->{ }
       ), debug: Crystal::Debug::All)
   end
+
+  it "doesn't fail on constant read calls (#11416)" do
+    codegen(%(
+      require "prelude"
+
+      STDERR.puts "false"
```

Comment:
A minimal reduction doesn't even need the call to `puts`. A program with only `STDERR` is sufficient.
```suggestion
      STDERR
```